test/lib/sdk/jsSdk.js

```diff
@@ -21,13 +19,16 @@ class JsSdk {
   exists() {
     const packageFile = `${this.sdkDir}/package.json`;
 
-    if (! fs.existsSync(this.sdkDir) || ! fs.existsSync(packageFile)) {
+    if (!fs.existsSync(this.sdkDir) || !fs.existsSync(packageFile)) {
       return false;
     }
 
     const versionString = JSON.parse(fs.readFileSync(packageFile))._id;
 
-    if (! versionString || ! versionString.includes(`kuzzle-sdk@${this.version}`)) {
+    if (
```

Comment:
(nitpicking) lines 28-35 can be simplified:

```js
return versionString && versionString.includes(`kuzzle-sdk@${this.version}`);
```

Reply:
Other than taking 2 `!` away from the code, I don't see the point of changing this. Not my code, btw. Ended here because of Prettier :woman_shrugging:

Reply:
This is a nitpicking issue, so it doesn't matter :+1: 
(it removes the entire `if (condition is false) { return true; } else { return false; }` condition)